src/styles/blocks/recomended.css

```diff
@@ -0,0 +1,43 @@
+.recomended__item {
+  margin-bottom: 40px;
+}
```

Comment:
Use SCSS instead of CSS for styling since the task specified to write styles in the `main.scss` file.